spec/gry/rubocop_adapter_spec.rb

```diff
@@ -115,4 +115,12 @@
       end
     end
   end
+
+  describe '.find_target_files' do
+    subject{Gry::RubocopAdapter.find_target_files}
+
+    it {is_expected.to be_a Enumerable}
+    it {is_expected.to be_all{|line| line.is_a? String}}
```

Comment:
__[rubocop]__
* Lint/AmbiguousBlockAssociation: Parenthesize the param `be_all{|line| line.is_a? String}` to make sure that the block will be associated with the `be_all` method call. [:ref](https://github.com/bbatsov/ruby-style-guide#syntax)

<!-- Comment by [SideCI](https://sideci.com) -->